packages/terra-time-input/tests/wdio/time-input-spec.js

```diff
@@ -0,0 +1,268 @@
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+});
+
+describe('Time Input onChange operations', () => {
+  describe('Time Input OnChange', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+      browser.keys('1234');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input prepends 0 on single digit hour', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+      browser.keys('2');
+      browser.click('#timeInput input[name="terra-time-minute-time-input"]');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input prepends 0 on hour input of 3 or more', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+      browser.keys('3');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input does prepends 0 on hour input less than 3', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+      browser.keys('2');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input does not accept hour input greater than 23', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+      browser.keys('24');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input prepends 0 on single digit minute', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-minute-time-input"]');
+      browser.keys('2');
+      browser.click('#timeInput input[name="terra-time-hour-time-input"]');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input prepends 0 on minute input of 6 or more', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-minute-time-input"]');
+      browser.keys('6');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Time Input does not prepends 0 on hour input less than 6', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/time-input/time-input/on-change');
+
+      browser.click('#timeInput input[name="terra-time-minute-time-input"]');
+      browser.keys('5');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
```

Comment:
I think these screenshots need checked out. Check out this corresponding screenshot: https://github.com/cerner/terra-core/blob/d5f60148c62709480244d36f679c960553207db8/packages/terra-time-input/tests/wdio/__snapshots__/reference/chrome/Time_Input_does_not_prepends_0_on_hour_input_less_than_6%5Bdefault%5D.838x768.png

Reply:
The test was renamed incorrectly. It should be referencing the minute, not the hour. I've updated and fixed it.